zeth-contracts/contracts/BaseMixer.sol

```diff
@@ -93,35 +99,99 @@ contract BaseMixer is MerkleTreeMiMC7, ERC223ReceivingContract {
     }
 
     // ============================================================================================ //
+    // Reminder: The primary inputs are either field elements, hash digest or binary arrays.
+    // As a field element may not represent an entire hash digest, we formatted the former in two parts:
+    // a field element and aggregated the remaining bits into (a) common field element(s): the blob(s).
```

Comment:
`common field element(s)`, common bits to what?
Please rephrase to clarify this explanation.

What you do here is basically pack a set of N hash_digest of length > field capacity into a set of N + M field elements. The first 253 (field capacity) bits of the digests are packed into a field element and the remaining bits (hash_digest_length - field_capacity bits) are packed independently into a set of M field elements